industrial_reconstruction/industrial_reconstruction/archive_player.py

```diff
@@ -1,16 +1,4 @@
-# Copyright 2022 Southwest Research Institute
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
+#!/usr/bin/python3
```

Comment:
Maintain original copyright and license preamble. Add copyright stating modifications by Cohesive Robotics